content/articles/android/index.md

```diff
@@ -135,6 +135,14 @@ An apk will be in `target/android-artifacts/debug/apk` or `target/android-artifa
 
 # 3. Fine tuning the game for android
 
+## Targeting API Level >= 31
```

Comment:
I believe cargo-apk should do it by itself and fix is on the way! 

but, for now, better have it in the docs, thank you for mentioning this!